xds/src/main/java/io/grpc/xds/internal/rbac/engine/cel/IncompleteData.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine.cel;
+
+/**
+ * Add an interface for interpreter to check if an object is an instance
```

Comment:
"Add an ..." sounds like the PR description copied as Javadoc comment here. Shouldn't this be something like "Interpreter returns an instance of `{@code IncompleteData}` if an object is an instance of `{@link PartialMessage}`" . 

I am assuming `PartialMessage` is defined somewhere.

Reply:
Thanks for the review! `PartialMessage` is defined internally, and from API-wise we don't need it at this moment. Regarding to the comment, We copied them from the internal CEL lib, but I agreed that the original comments could be improved, and yours are much better... @cindyxue Can you please update them? Thanks!

Reply:
Thank you both for the review and the comments! Fixed :)